### PR TITLE
Doctest

### PR DESCRIPTION
**Pull request by Andrey-Raspopov**
This pull request addresses issue #1902 and doctest for module BioSql.BioSeqDatabase.

- [ X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [ X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [ X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
Note there is a flake8 version conflict right now, see https://gitlab.com/pycqa/flake8-docstrings/issues/36

**Comment by peterjc** [comment]
This looks sensible :)

**Comment by Andrey-Raspopov** [comment]
Yup, I saw that last several builds were unsuccesfull.

File: Tests/run_tests.py [hunk 1]
```diff
@@ -91,8 +91,7 @@ def is_pypy():
     'Bio.Phylo.Applications._Fasttree',
     'Bio.Phylo._io',
     'Bio.Phylo.TreeConstruction',
-    'Bio.Phylo._utils',
-    'BioSQL.BioSeqDatabase',
+    'Bio.Phylo._utils'
```
Comment (peterjc) [inline Tests/run_tests.py]: Could you replace the trailing comma on the line above?

This is a minor style point, but the idea here is then adding/removing entries only affects one line in the diff. There is even a flake8 plugin to enforce trailing commas.

**Comment by peterjc** [comment]
It looks like some of your commits were done under a different email address, GitHub has only linked your account to one of them. If I squash merge this, it might pick the first email address. Do you care about the exact attribution?

In any case, I've suggested restoring a comma - doing another commit should give us a fresh go at TravisCI since the problem on the master has been fixed now.

File: Tests/run_tests.py (hunk 1 above)
Comment (Andrey-Raspopov) [reply Tests/run_tests.py]: I've changed it, yeah. Couldn't put my finger on the problem here.

**Comment by Andrey-Raspopov** [comment]
This is the same e-mail, but it was pushed from the remote machine, without my keys, so it may be the problem. You can squash it without a doubt:)

**Comment by peterjc** [comment]
Thank you.

P.S. The final squashed commit 568b429174c2c37da50b2d6ba251bf85a8923f00 does seem to be linked to your GitHub profile :)

**Pull request merged**